custom_components/tuya_local/devices/nince_rgb_led_strip.yaml

```diff
@@ -0,0 +1,158 @@
+            4646020032025803e800000000505002001e02ee03e8000000003232020000\
+            03e803e800000000"
+          value: "Sunset glow"
+        - dps_val: "1746460200f003e803e80000000046460200dc02bc03e800000000"
+          value: "Ocean"
+        - dps_val: "184646020028032003e800000000464602001e038403e800000000\
+            4646020014038403e800000000"
+          value: "Sunflower"
+        - dps_val: "19464601007803e803e800000000464602006e0320025800000000\
+            464602005a038403e800000000"
+          value: "Forest"
+        - dps_val: "1a464602000a038403e800000000464602000003e803e800000000"
+          value: "Kung Fu"
+        - dps_val: "1b464603001803e803e800000000"
+          value: "Candlelight"
+        - dps_val: "1c4646020104032003e800000000464602011802bc03e800000000\
+            464602011303e803e800000000"
+          value: "Dream"
+        - dps_val: "1d646401000003e803e80000000064640100f003e803e800000000\
+            646402007803e803e800000000646402003d03e803e800000000"
+          value: "Mediterranean"
+        - dps_val: "1e323201015e01f403e800000000323202003201f403e800000000\
+            32320200a001f403e800000000"
+          value: "French Street"
+        - dps_val: "1f46460100dc02bc03e800000000464602006e0320025800000000\
+            4646020014038403e800000000464601012703e802ee000000004646010000\
+            0384028a00000000"
+          value: "American"
+        - dps_val: "20646401003d03e803e800000000646401007803e803e800000000\
+            5a5a01011303e803e8000000005a5a0100ae03e803e8000000006464010032\
+            01f403e800000000646401000003e803e800000000"
+          value: "Birthday"
+        - dps_val: "21323202015e01f403e800000000323202011303e803e800000000"
+          value: "Wedding day"
+        - dps_val: "225a5a0100f003e803e8000000005a5a01003d03e803e800000000\
+            464601000003e803e8000000005a5a0100ae03e803e8000000005a5a010113\
+            03e803e800000000464601007803e803e800000000"
+          value: "Christmas"
+        - dps_val: "23505002000003e803e80000000046460200f003e803e800000000"
+          value: "Independence day"
+        - dps_val: "24464602000003e803e800000000464602003d03e803e800000000\
+            464602011303e803e80000000046460200f003e803e8000000004646020078\
+            03e803e800000000"
+          value: "Diwali"
+        - dps_val: "25464601011303e803e800000000464602000003e803e800000000\
+            464602003d03e803e8000000004646010154032003e8000000004646010140\
+            032003e800000000464601001e02ee03e800000000"
+          value: "Holi"
+        - dps_val: "265a5a020014006403e800000000464602000003e803e800000000"
+          value: "Victory Day"
+        - dps_val: "275a5a020014006403e800000000464602000003e803e800000000\
+            323202015e01f403e800000000464602011303e803e800000000"
+          value: "Easter"
+        - dps_val: "28464601011303e803e800000000464601001e03e803e800000000"
+          value: "Halloween"
+    - id: 22
+      name: brightness
```

Comment:
This is duplicated with the dp 24 brightness below. If the light strip only supports controlling via dp 24, then I think removing colour_mode and BOTH brightness dp definitions should work.  The bulb will then always be in hs mode (which I think is correct, even when a scene is selected), and the code that automatically controls brightness through the v parameter of rgbhsv will be used.